packages/gatsby/src/services/initialize.ts

```diff
@@ -65,20 +65,11 @@ if (
   sampleSiteForExperiment(`DEV_SSR`, 5)
 ) {
   showExperimentNoticeAfterTimeout(
-    `devSSR`,
-    `
-Your dev experience is about to get better, faster, and stronger!
+    `SSR in Development`,
```

Comment:
Not hill I would die on, but if we can let's try to not force users to google acronyms ;)
```suggestion
    `Server Side Rendering (SSR) in Development`,
```